README.md

```diff
@@ -22,6 +22,68 @@ To get started with the D4 Dashboard, you'll need to have [node.js](https://node
 
 To start the app in development mode, run `npm start`. This will launch the app in your default browser, where you can begin using it.
 
+## Setting Up Your `.env` File in Strapi
```

Comment:
Also add 
`
## Setting Up Your `.env`

### Step 1: Open your terminal and navigate to the root directory of this project

```bash
cd path/to/thisproject
```

### Step 2: Create the .env file.

```bash
touch .env
```
### Step 3: Open the .env file with your preferred text editor and add the following content.
```REACT_APP_STRAPI_URL=https://pitne-d4-app-strapi-production.up.railway.app
REACT_APP_STRAPI_USER=username
REACT_APP_STRAPI_PASSWORD="password"
```


### Step 4: Fill in your values by replacing placeholders with your actual values.


### Step 4: Save and rebuild the app.
`

Reply:
I have included the above text in the next commit, updating the instructions for the .env file in the root directory.